cypress_tests/cypress/integration/quiz/quiz_with_logged_in_user.spec.js

```diff
@@ -0,0 +1,21 @@
+describe("Quiz with logged in user tests", () => {
+    const url = "/en/sections/questionnaire-testing/sample-quiz/";
+
+    before("Login", () => {
+        cy.login("saqlain", "saqlain");
```

Comment:
Could you add another test where no user is logged in, to check that the "Login to participate" button is shown? Note that this was only merged in today, so you might have to merge/rebase onto develop.